lib/flame/controller/actions.rb

```diff
@@ -60,11 +59,12 @@ def inherit_actions(actions = superclass.actions, exclude: [])
 			##   end
 			def with_actions(mod, exclude: [], only: nil)
 				Module.new do
-					@mod = mod
-					@methods_to_define =
-						only || (@mod.public_instance_methods(false) - exclude)
-
-					extend ModuleWithActions
+					define_singleton_method(:included) do |target|
+						target.send(
+							:copy_actions_from, mod,
+							only || mod.public_instance_methods(false) - exclude
+						)
```

Comment:
I don't like sending to methods that can be not private. Can it be `protected`? If not — let's make it `public`.

Reply:
It's replaced with changed `inherit_actions` now.